code/tools/vboreplay/src/VBOReplay.cpp

```diff
@@ -60,22 +60,19 @@ void VBOReplay::setUp()
   if (!(LAT < 0) && !(LON < 0)) {
     // NORTH/WEST
     ref =
-    WGS84Coordinate(LAT, WGS84Coordinate::NORTH, LON, WGS84Coordinate::WEST);
+    WGS84Coordinate(LAT, LON);
```

Comment:
Hi @bjornborg, this should be WGS84Coordinate(LAT, LON); only; you can remove the entire If(...) statements.